### DISCUSSION TITLE
Mod的简体中文翻译 累死我了！翻译还不完善

### DISCUSSION THREAD

**Discussion by QwQ-zhuoze** [thatskymod/Sky-CotL-Scripts#118] [Show and tell]
![image](https://github.com/user-attachments/assets/aed528cc-13e2-4b4a-8527-82e5c4f5545f)
一部分是机翻，很多都是自己亲自翻译亲自试的

**Comment by nick0414056** [comment]
可以分享一下如何使用嗎 我遇到很多問題

**Reply by QwQ-zhuoze** [reply]
您好！你可以查看我的分支仓库，里面有我写的教程和文件。有任何疑问可以随时问我，我看见都会回复的！
这个是地址：https://github.com/QwQ-zhuoze/Sky-CotL-Scripts-Chinese
如果有需要我还会出繁体版本，谢谢

**Reply by nick0414056** [reply]
謝謝妳 那這有steam 端的嗎?
